samples/README.md

```diff
@@ -8,8 +8,12 @@
 ```
 oc create secret generic github-secret --from-literal=GIT_USER=<username> --from-literal=GIT_TOKEN=<personal access token> --namespace pelorus
 ```
-4. Run `helm install operators charts/operators --namespace pelorus`
-5. Update the values.yaml file with the following:
+4. Create a build webhook secret: 
+```
+oc create secret generic mongodb-build-webhook --from-literal=WebHookSecretKey=<secret>
```

Comment:
this is related to one particular project: mongodb-build-webhook, where other examples are generic. How about add <app-name-webhook> and then example mongodb-build-webhook ?